tox.ini

```diff
@@ -43,6 +43,7 @@ deps =
     flake8-docstrings-complete>=1.2.0
     flake8-test-docs>=1.0
     isort
+    mock
```

Comment:
What is the reason for needing this please?

Reply:
pylist complains about not being able to import this package when linting

Reply:
Isn't mock in the inbuilt unittest library? This sounds like something we should investigate - mock shouldn't need to be pip installed

Reply:
It is, but we are not using unittest here

Reply:
The `mock` library is a backport to previous version of python of the `unittest.mock` library. Unless you need the backport from a later version of python, importing `unittest.mock` is equivalent to installing `mock` regardless of whether pytest is being used for testing. you can use `unittest.mock` in pytest. Maybe I'm misunderstanding something?

Reply:
Yes, I can. I think it was me who misunderstood